mmv1/templates/terraform/examples/gkehub_fleet_basic.tf.erb

```diff
@@ -1,3 +1,9 @@
 resource "google_gke_hub_fleet" "default" {
   display_name = "my production fleet"
+  default_cluster_config {
```

Comment:
This is a docs-only example; usually we restrict the basic test to only cover required fields - just want to double-check that this is expected to be in most configurations?

Reply:
Yes, I think this is the most likely configuration for users using terraform. Defining the fleet resource in terraform is generally worthless without using the default cluster config.